### ISSUE TITLE
Cleanup serialization of ECS world, commands, and events

### ISSUE THREAD

**Issue by rpadrela** [coherence/replication-server#197]
**Background**

It's hard to work on, it's hard to read and it would be good to remove the budgets to make 100% use of a packet.

**Expected**

- Clearer and more concise names
- No magic numbers
- Removal of the various budgets, if possible

**Comment by rpadrela** [comment]
@cbrisebois , I think you've done some of this (removing the budgets)?

**Comment by rpadrela** [comment]
> @cbrisebois , I think you've done some of this (removing the budgets)?

Changed the task as the budgets have been removed.

**Comment by cbrisebois** [comment]
yeah, but I didn't rename everything.  I tried to change names to write* instead of send* when it was writing to a stream (serializing).  I'm sure there's more there to clean up.
